body.tex

```diff
@@ -242,10 +251,15 @@ \section*{Interoperability and extensibility}
 
 Subclasses of NumPy often exist due to difficulty in constructing richer data
 types, such as quantities with physical units \cite{astropy,Goldbaum2018,pint},
-geometrical objects \cite{pygeos}, and missing numbers.
+geometrical objects \cite{pygeos}, and missing numbers. % pandas: categoricals, IP addresses?
 To improve \emph{extensibility}, we are currently overhauling the data type
 system to make it easier to implement data types both in Python and C, and to
 support these and other applications.
+The widespread adoption of rich, user defined, data types can allow more
+concise, feature rich scientific computing.
+By making these user implemented data types a fundamental design of NumPy
+these tools will enabled and interact naturally with different libraries.
```

Comment:
maybe a bit of clarity on what "enable and interact naturally with different libraries" means.

Reply:
Yeah, I would think of 1. I expect multiple user dtypes can interact (naturally); 2. libraries such as sklearn (or dask) might "just work" with e.g. physical Units (and Categoricals, which they probably really would like to use e.g. for labels); 3. Things being inside a dtype rather than subclass is a better user experience.
I agree its phrased much too fuzzy, but can't think of an improvement quickly right now, maybe the solution is just to expand it to a few sentences though.

Reply:
That sounds reasonable and I agree that you should expand this into a few sentences or more.